src/java.base/linux/native/libjava/CgroupMetrics.c

```diff
@@ -43,3 +44,12 @@ Java_jdk_internal_platform_CgroupMetrics_getTotalMemorySize0
     jlong page_size = sysconf(_SC_PAGESIZE);
     return pages * page_size;
```

Comment:
Preexisting, and theoretical nitpickery: I got curious when I saw this. What does sysconf return if a 32-bit VM runs on a large machine with >8Tb main memory and 4k pages? In that case, _SC_PHYS_PAGES would overflow the 32-bit long return type of sysconf.

Reply:
Yes. Any suggestions as to how to fix it? This code was actually changed recently with [JDK-8300119](https://bugs.openjdk.org/browse/JDK-8300119). Anyway, this should go into a separate bug if we change it. Do you agree?

Reply:
I honestly have no idea of how to fix it. A separate RFE would be fine.